client/pokemon/pokemon-full.view.html

```diff
@@ -25,8 +25,8 @@ <h1 class="summary-toolbar">{{pokemon.parsedNickname}}</h1>
       </div>
       <span flex></span>
       <a ng-href="/api/v1/pokemon/{{pokemon.id}}/raw">
-        <md-button class="md-icon-button" aria-label="Download .pk6 file" ng-show="pokemon.data.owner === main.user.name || main.user.isAdmin || pokemon.data.visibility === 'public'">
-          <md-tooltip md-direction="left">Download .pk6 file</md-tooltip>
+        <md-button class="md-icon-button" aria-label="Download file" ng-show="pokemon.data.owner === main.user.name || main.user.isAdmin || pokemon.data.visibility === 'public'">
+          <md-tooltip md-direction="left">Download file</md-tooltip>
```

Comment:
Instead of dropping the generation entirely, we could have it display the correct generation with something like this:

`Download .pk{{pokemon.data.gen}} file`

But I believe old files wouldn't have the gen unless we plan to do a migration. @not-an-aardvark, @SnowPhoenix?

(If we're not doing a migration, `Download .pk{{pokemon.data.gen || '6'}} file` would work.)